projects/plugins/super-cache/.phpcs.dir.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
```

Comment:
FYI, this new file will be included in the plugin zip. If you don't want it there (since it probably shouldn't be), you should add a line like this
```
.phpcs.dir.xml    production-exclude
```
to `projects/plugins/super-cache/.gitattributes`

Reply:
Ah, thanks for pointing this out. I've added a PR for that. https://github.com/Automattic/jetpack/pull/27251